### ISSUE TITLE
Add more database options

### ISSUE THREAD

**Issue by omwah** [omwah/SDFEconomy#1]
Add support for SQLite and MySQL storage options.

**Comment by RiflemanSD** [comment]
Hello @omwah .

This plugin is very nice. But there is a problem.
You are using only YAML for storing data, and that make a lot of lag when the yaml is big!
You must add SQL support.

And if u want, i can help you, i am creating a SQLite small lib now, and i can add it to your source, to support a SQLite storage.

**Comment by omwah** [comment]
Feel free to implement it and issue a pull request. If you want to become a maintainer then let me know. The other guys who mentioned working on the plugin seem to have gone inactive, at least on SDFEconomy.

**Comment by thesbros** [comment]
@omwah Ah, I don't develop for Minecraft anymore. You can remove me as a collaborator.

**Comment by omwah** [comment]
@thesbros Same here! No problem of having moved on, I have as well.

**Comment by RiflemanSD** [comment]
@omwah Yeah, i would like to be a maintainer of this project ! I have notice there are some improvements can be done here.
Also, i will do the pull request, after finish the lib and testing it.
Thanks !

**Comment by omwah** [comment]
Don't worry about a pull request, I just gave you commit privileges. You can commit directly.

**Comment by RiflemanSD** [comment]
@omwah Just to know, i had a problem with SQLite library, i am trying to fix it. But because i have a work that days, maybe i will late a little.
I will give you more info soon.

**Comment by omwah** [comment]
No problem
